### ISSUE TITLE
CLI: exit process with correct error code

### ISSUE THREAD

**Issue by aledeg** [danielgtaylor/aglio#303]
I've seen that there is already a resolved ticket (#38) about that issue. But it seems that when I have warnings during the generation, the return code is 0.

Here is the launched command:
```
aglio --theme flatly --theme-template index.jade --theme-full-width true -i book.apib -o book.html
```

Am I missing something?

**Comment by aledeg** [comment]
If I redirect stderr to a file, I get all errors and warnings in that file. That means that there is an unconsitency between the output and the exit code.
As a workaround, I output every error and warning messages to a file and test the file.

**Comment by jackbentley** [comment]
A warning message isn't an error, though? So it shouldn't affect the exit code? A non-zero exit code would be considered an error.

**Comment by aledeg** [comment]
I agree that a warning is not an error, but there should be something to differentiate a process that works flawlessly and one with warnings.
Maybe a switch to convert warnings to errors.
